nucliadb_protos/resources.proto

```diff
@@ -201,6 +201,7 @@ message ExtractedVectorsWrapper {
         CloudFile file = 2;
     }
     FieldID field = 3;
+    string vectorset_id = 4;
```

Comment:
Since `ExtractedVectorsWrapper` message is already prepared to be in a cloud file if it's too big, I think it'd be enough if we add the `vectorset_id` to it to be able to distinguish the different embedding models.